lib/puppet/functions/archive/go_md5.rb

```diff
@@ -0,0 +1,36 @@
+require_relative '../../../puppet_x/bodeco/util.rb'
+
+# @summary
+#   Retrieves and returns specific file's md5 from GoCD server md5 checksum file
+# @api private
+# @see http://www.thoughtworks.com/products/docs/go/12.4/help/Artifacts_API.html
```

Comment:
I took this 'broken' link directly from the old function.  I was going to update with eg. https://api.gocd.org/current/#get-artifact-file, but that says `Available since v14.3.0`.  This is why I suspect this function (and `archive::go`) might be outdated and of limited value until a GoCD user comes along and fixes them up.